### PR TITLE
Add Hello in many more Indigenous languages.

### PR DESCRIPTION
**Pull request by erinishimoticha**

### CODE REVIEW

File: src/components/Home.re [hunk 1]
```diff
@@ -255,6 +255,15 @@ let make = () => {
                 "Hesci",
                 {j|Yá'át'ééh|j},
                 "Halito",
+                {j|ᎣᏏᏲ|j},
+                "Aaniin",
+                "Wáa sá i yatee",
```
Comment (arecvlohe) [inline src/components/Home.re]: Have to use `{j||j}` here as well for those special characters

File: README.md [hunk 2]
```diff
@@ -39,7 +39,8 @@ Packages are managed using [yarn](https://yarnpkg.com/en/) for more deterministi
 Running the application locally requires a few steps.
 
 - Clone this repository to your desktop (or somewhere else on your computer): `git clone git@github.com:nativesintech/natives-in-tech.git`
-- From the terminal, install dependencies: `$ yarn install`
+- From the terminal, install local dependencies: `$ yarn install`
+- From the terminal, install global dependencies: `npm i -g yarn phenomic npm-run-all`
```
Comment (arecvlohe) [inline README.md]: Thanks!

Comment (arecvlohe) [reply README.md]: Should an `.nvmrc` be added?

**Comment by arecvlohe** [comment]
@all-contributors please add @erinishimoticha for code,doc

**Comment by arecvlohe** [comment]
@all-contributors please add @erinishimoticha for code and doc

**Comment by erinishimoticha** [comment]
Updated that string; added a missing global dependency to the README.

Test evidence: https://www.youtube.com/watch?v=s8OPC7KSAdI&feature=youtu.be

File: src/components/Home.re [hunk 3]
```diff
@@ -250,11 +250,20 @@ let make = () => {
...
+                "Halito",             // Choctaw
+                {j|ᎣᏏᏲ|j},            // Cherokee
+                "Aaniin",             // Anishinaabemowin
+                {j|Wáa sá i yatee|j}, // Tlingit
+                "Pablan",             // Inupiaq
+                "Sekoh",              // Mohawk / Kanien'keha
+                "Howa",               // Osage
+                "Ba'ax ka wa'alik",   // Mayan
+                "Niltse",             // Nahuatl
+                "Tanisi",             // Cree
```
Comment (arecvlohe) [inline src/components/Home.re]: 👍

**Pull request merged**